app/test/qea-property-analyzer-test.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../../.tmp/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../.tmp/bower_components/web-component-tester/browser.js"></script>
+    <!-- import the element to test -->
+    <link rel="import" href="../../.tmp/elements/qea-property-analyzer/qea-property-analyzer.html">
+  </head>
+  <body>
+    <!-- use the document as a place to set up your fixtures -->
+    <test-fixture id="qea-property-analyzer-fixture">
+      <template>
+        <qea-property-analyzer ></qea-property-analyzer>
+      </template>
+    </test-fixture>
+    <script>
+      suite('qea-property-analyzer test', function() {
+        var myEl, listItems;
+
+        setup(function() {
+          myEl = fixture('qea-property-analyzer-fixture');
+          stub('qea-property-analyzer', {
+            resolvePath: function(tagName) {
+              return '../../.tmp/bower_components/'+tagName+'/'+tagName+'.html'
+            }
+          });
+        });
+
+
+        test('test scanElemen method', function(done) {
```

Comment:
Your don't need "check" or  "test" on your test messages, it is implied.  You should try to thing of the message like :
(test that the) "scanElement method succeeds for asq elements'".  Check here for some examples: https://github.com/Polymer/designer/blob/master/test/path/path_test.html